typespec-tests/src/test/java/com/_specs_/azure/clientgenerator/core/internal/InternalTests.java

```diff
@@ -1,49 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com._specs_.azure.clientgenerator.core.internal;
-
-import com.azure.core.http.rest.RequestOptions;
-import com.azure.core.http.rest.Response;
-import com.azure.core.util.BinaryData;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
-
-import java.util.HashMap;
-import java.util.Map;
-
-public class InternalTests {
```

Comment:
@XiaofeiCao 

I think Alan deleted it because there is large change on cadl-ranch test case on internal. Would you add it back later with the updated tests?

Reply:
Yes, the case was what this was calling into changed, failed compile, and didn't have a replacement for the calls it was making before.